Tools/Scripts/webkitpy/layout_tests/run_webkit_tests_integrationtest.py

```diff
@@ -732,6 +732,18 @@ def test_crash_log(self):
         expected_crash_log = mock_crash_report
         self.assertEqual(host.filesystem.read_text_file('/tmp/layout-test-results/failures/unexpected/crash-with-stderr-crash-log.txt'), expected_crash_log)
 
+    def test_crash_log_webkit_test_runner(self):
+        # FIXME: Need to rewrite these tests to not be mac-specific, or move them elsewhere.
+        # Currently CrashLog uploading only works on Darwin and Windows.
```

Comment:
Not sure why this has to test through crash log path, this seems worth explaining in ChangeLog.